### ISSUE TITLE
BUG REPORT:  Tool Changes Camera Ip to an unreachable address ; cannot run its tests

### ISSUE THREAD

**Issue by dwilliamsND** [BaslerSupportUSA/CQT#2]
** Summary **
The 'CameraQuickTester' leaves the camera in a broken state (fixable, but broken) and fails to run any tests.

**Priority**
Entirely breaks functionality, but for a unknown % of users.

**To Reproduce**
(Always happens on my workstation ... fwiw)
1. Run tool `CameraQuickTester` from the command line, connect to any of the available cameras

** Actual Behavior ** 
1. Run tool with: `CameraQuickTester -serialnumber 23046693 -grabtimesec 10`
2. Tool connects to camera on a known ip address
3. Tool detects an "IP Address Incompatibility"
4. Tool Changes camera ip address to 255.255.255.130
5. Tool cannot reconnect to camera
6. Tool exits

**Expected behavior**
1. If Tool Connects to the camera, no network changes are made
2. Tool proceeds to run its tests as normal.

**CQT Files**
[Step_000_Advice_23046693_2020-02-07_0924AM.txt](https://github.com/BaslerSupportUSA/CQT/files/4171503/Step_000_Advice_23046693_2020-02-07_0924AM.txt)
[Step_000_Log_23046693_2020-02-07_0924AM.txt](https://github.com/BaslerSupportUSA/CQT/files/4171504/Step_000_Log_23046693_2020-02-07_0924AM.txt)

**CQT Version**
20.02-01

**Pylon Version**
5.2.0.13457

**OS Version**
Ubuntu 16.04 LTS

**Camera Version**
acA1300-75gc

** Network Setup ** 
- Camera IP: 192.168.3.52
- (multiple switches)
- local network configuration: (excerpt from /etc/network/interfaces; OS: Ubuntu 16.04 LTS)
```
auto enp0s31f6
iface enp0s31f6 inet dhcp

auto enp0s31f6:0
iface enp0s31f6:0 inet static
     address 192.168.3.19
     netmask 255.255.255.0
```
User's workstation has an unusuable network configuration -- Two addresses bound to the same NIC -- but workstation may otherwise communicate with the cameras, without issue.

**Comment by dwilliamsND** [comment]
ifconfig output, on the problematic workstation:
```
[09:48:25]#>lsb_release -d
Description:	Ubuntu 16.04.6 LTS
[09:42:03]#>ifconfig 
enp0s31f6 Link encap:Ethernet  HWaddr d4:81:d7:d1:a0:e9  
          inet addr:172.18.44.166  Bcast:172.18.44.255  Mask:255.255.255.0
          inet6 addr: fe80::d681:d7ff:fed1:a0e9/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:27268 errors:0 dropped:0 overruns:0 frame:0
          TX packets:19274 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:18068903 (18.0 MB)  TX bytes:4463487 (4.4 MB)
          Interrupt:16 Memory:dd400000-dd420000 

enp0s31f6:0 Link encap:Ethernet  HWaddr d4:81:d7:d1:a0:e9  
          inet addr:192.168.3.19  Bcast:192.168.3.255  Mask:255.255.255.0
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          Interrupt:16 Memory:dd400000-dd420000 

lo        Link encap:Local Loopback  
          inet addr:127.0.0.1  Mask:255.0.0.0
          inet6 addr: ::1/128 Scope:Host
          UP LOOPBACK RUNNING  MTU:65536  Metric:1
          RX packets:238 errors:0 dropped:0 overruns:0 frame:0
          TX packets:238 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:18294 (18.2 KB)  TX bytes:18294 (18.2 KB)

wlp2s0    Link encap:Ethernet  HWaddr 14:ab:c5:c6:66:b9  
          UP BROADCAST MULTICAST  MTU:1500  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)

wlp2s0:avahi Link encap:Ethernet  HWaddr 14:ab:c5:c6:66:b9  
          inet addr:169.254.10.189  Bcast:169.254.255.255  Mask:255.255.0.0
          UP BROADCAST MULTICAST  MTU:1500  Metric:1
```

**Comment by dwilliamsND** [comment]
This issue seems fixed by the latest release: v20.03-1(Beta).

**Comment by BaslerSupportUSA** [comment]
sorry for the delay, but thanks for submitting the issue! glad the fix worked!
-matt
